src/components/VersionChooser/index.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { Col, Form, Row } from 'react-bootstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import VersionSelect, { Version } from '../VersionSelect';
+import { gettext } from '../../utils';
+import styles from './styles.module.scss';
+
+type PublicProps = {
+  versions: Version[];
+};
+
+class VersionChooserBase extends React.Component<PublicProps> {
+  render() {
+    const { versions } = this.props;
+
+    const listedVersions = versions.filter(
+      (version) => version.channel === 'listed',
+    );
+    const unlistedVersions = versions.filter(
+      (version) => version.channel === 'unlisted',
+    );
```

Comment:
The two `filter()` calls might go to the reducer in the near future and from there, we could have an enum or constants to represent the `channel` of a version. It's either `listed` or `unlisted` AFAIK.